compiler/platform.nim

```diff
@@ -208,7 +213,8 @@ const
     (name: "sparc64", intSize: 64, endian: bigEndian, floatSize: 64, bit: 64),
     (name: "mips64", intSize: 64, endian: bigEndian, floatSize: 64, bit: 64),
     (name: "mips64el", intSize: 64, endian: littleEndian, floatSize: 64, bit: 64),
-    (name: "riscv64", intSize: 64, endian: littleEndian, floatSize: 64, bit: 64)]
+    (name: "riscv64", intSize: 64, endian: littleEndian, floatSize: 64, bit: 64),
+    (name: "armv8a57", intSize: 64, endian: bigEndian, floatSize: 64, bit: 64)]
```

Comment:
Can't ``arm64`` be reused? Is there a specific reason you need ``bigEndian``?

Reply:
Yep, you're right. Checked the switch's cpu mode and it can be either.